sources/packages/web/src/services/http/dto/Institution.dto.ts

```diff
@@ -37,6 +38,7 @@ export interface InstitutionProfileAPIInDTO extends InstitutionContactAPIInDTO {
   primaryEmail: string;
   website: string;
   regulatingBody: string;
+  otherRegulatingBody: string;
```

Comment:
`otherRegulatingBody?: string;` as its optional

Reply:
Sounds good. Will do the change.